### PR TITLE
Add messaging health dependency to bom

### PR DESCRIPTION
**Pull request by danielkec**
Signed-off-by: Daniel Kec <daniel.kec@oracle.com>

### CODE REVIEW

File: docs/mp/reactivemessaging/01_introduction.adoc [hunk 1]
```diff
@@ -161,3 +161,34 @@ Declare the following dependency in your project:
...
+   <groupId>io.helidon.microprofile.messaging</groupId>
+   <artifactId>helidon-microprofile-messaging-health</artifactId>
+</dependency>
+----
+
+* Liveness - channel is considered UP until `cancel` or `onError` signal is intercepted on it.
+* Readiness - channel is considered DOWN until `onSubscribe` signal is intercepted on it.
+
+
+If you check you health endpoints `/health/live` and `/health/ready` you will discover
```
Comment (ljamen) [inline docs/mp/reactivemessaging/01_introduction.adoc:180]: your

**Review by ljamen — APPROVED** [review]
Good just a single typo

File: docs/mp/reactivemessaging/01_introduction.adoc (hunk 1 above)
Comment (Verdent) [inline docs/mp/reactivemessaging/01_introduction.adoc:180]: Typo

> If you check your

**Review by Verdent — CHANGES_REQUESTED** [review]

File: docs/mp/reactivemessaging/01_introduction.adoc (hunk 1 above)
Comment (danielkec) [reply docs/mp/reactivemessaging/01_introduction.adoc]: thx

Comment (danielkec) [reply docs/mp/reactivemessaging/01_introduction.adoc]: thx

**Pull request merged**